libs/cardano-protocol-tpraos/CHANGELOG.md

```diff
@@ -1,9 +1,22 @@
 # Version history for `cardano-protocol-tpraos`
 
-## 1.0.1.1
+## 1.0.3.2
 
 *
 
+## 1.0.3.1
+
+* upper bounds on cardano-ledger-core and cardano-ledger-shelley
+
+## 1.0.3.1
```

Comment:
There is also no `1.0.3.1` release, but there is `1.0.3.0`, which is not in the changelog:
```suggestion
## 1.0.3.0
```

Reply:
`1.0.3.1` is what I am trying to release now: https://github.com/input-output-hk/cardano-haskell-packages/pull/319. so `1.0.3.2` is next smallest next bump.

This changelog seems to have been neglected, so I dug through the git history, but maybe I made a mistake. I think the real problem is that I listed `1.0.3.1` twice. the "lower bound" entry should be under `1.0.3.0`.